io.github.wereturtle.ghostwriter.json

```diff
@@ -0,0 +1,120 @@
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "cmark",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/commonmark/cmark.git",
+                    "tag": "0.28.3"
+                }
+            ]
+        },
+        {
+            "name": "cmark-gfm",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/github/cmark.git",
+                    "tag": "0.28.3.gfm.12"
+                }
+            ]
+        },
+        {
+            "name": "MultiMarkdown",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/fletcher/MultiMarkdown-6.git",
+                    "tag": "6.3.2"
+                }
+            ]
+        },
+        {
+            "name": "discount",
+            "no-autogen": true,
+            "sources":[
+                {
+                    "type":"archive",
+                    "url": "https://github.com/Orc/discount/archive/v2.2.3a.tar.gz",
+                    "sha256": "40defe460dc005dbc78a4e6174fce29fca2ac3ccd5acd400c05eb713f824df97"
+                },
+                {
+                    "type": "shell",
+                    "commands": ["./configure.sh --prefix=/app"]
+                }
+            ]
+        },
+        {
+            "name": "Pandoc",
+            "only-arches": ["x86_64"],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp bin/pandoc /app/bin/pandoc",
+                "cp bin/pandoc-citeproc /app/bin/pandoc-citeproc"
+            ],
+            "sources":[
+                {
+                    "type":"archive",
+                    "url": "https://github.com/jgm/pandoc/releases/download/2.2.1/pandoc-2.2.1-linux.tar.gz",
+                    "sha256": "3b1b573cdf957b4102c42ce6f5a641787267a55a5e88dcca942fd471c25793cb"
+                }
+            ]
+        },
+        {
+            "name": "ghostwriter",
+            "buildsystem": "simple",
```

Comment:
flatpak-builder has also `qmake` buildsystem since a while.

Reply:
Unfortunately, some additional steps are required for pulling in qtwebkit, so building using the `qmake` buildsystem isn't possible.  Unless you have a suggestion?

Reply:
Let's solve this one and it's good to merge from my perspective.

Reply:
Okie dokie.  Yes, it looks much cleaner with the qmake build system.  About 2 months ago, the only thing that worked with QtWebKit was the CuteMarkEd way.  But apparently updating to a newer (latest) version of Flatpak seems to have fixed the issues I was having.  Thanks!